src/rpc-browser-ws.ts

```diff
@@ -2,15 +2,22 @@
 // see: https://github.com/microsoft/TypeScript/issues/16577#issuecomment-343610106
 
 import { RpcCommon, IPromiseCallbacks } from "./rpc-common.js";
+import { IChildLogger } from "@vscode-logging/types";
+import { noopLogger } from "./noop-logger";
 
 export class RpcBrowserWebSockets extends RpcCommon {
+  private static readonly className = "RpcBrowserWebSockets";
+
   ws: WebSocket;
+  logger: IChildLogger;
```

Comment:
Add "private readonly".
Sorry I didn't say before - just saw it in yeoman-ui now.

Reply:
done